python/sglang/srt/constrained/fsm_cache.py

```diff
@@ -21,7 +21,27 @@ def __init__(self, tokenizer_path, tokenizer_args_dict, enable=True):
             tokenizer = AutoTokenizer.from_pretrained(
                 tokenizer_path, **tokenizer_args_dict
             )
-            self.outlines_tokenizer = TransformerTokenizer(tokenizer)
+            try:
+                self.outlines_tokenizer = TransformerTokenizer(tokenizer)
+            except AttributeError:
```

Comment:
When the error is not about `pad_token_id`, we may still raise agian.

Reply:
The `AttributeError` seems only happen for `pad_token_id` in the TransformerTokenizer initialization.